recipes-debian/libpam-radius-auth/libpam-radius-auth_debian.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "This is the PAM to RADIUS authentication module."
+LICENSE = "GPLv2+ & Livingston"
+LIC_FILES_CHKSUM = "file://debian/copyright;md5=fe546348781fcbc12a5e7b8ae6414e60"
+NO_GENERIC_LICENSE[Livingston] = "debian/copyright"
+
+# Set DEBIAN_UNPACK_DIR to the path to the extracted Debian source package.
+DEBIAN_UNPACK_DIR = "${WORKDIR}/pam_radius-release_1_4_0"
+
+inherit autotools-brokensep debian-package
+require recipes-debian/sources/libpam-radius-auth.inc
+
+DEPENDS += "libpam"
+REQUIRED_DISTRO_FEATURES = "pam"
+
+FILES_${PN} += "${base_libdir}/security/* ${sysconfdir}/* ${datadir}/* ${docdir}/*"
+
+do_install() {
+    install -d ${D}${sysconfdir}
+    install -m 600 ${S}/pam_radius_auth.conf ${D}${sysconfdir}
+    install -d ${D}${base_libdir}/security
+    install -m 755 ${S}/pam_radius_auth.so ${D}${base_libdir}/security
+    install -d ${D}${datadir}/lintian/overrides
```

Comment:
lintian is debian package checker so this directory and file aren't needed on a target board , are it?

Reply:
As you said, I removed it.